WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -62,6 +62,8 @@
     <span>PID:</span> <%= Process.pid %>
     <%= icon "circle" %>
     <span>Uptime:</span> <%= duration_to_s(Time.now - WcaOnRails::BOOTED_AT) %>
+    <%= icon "circle" %>
+    <a href="https://github.com/thewca/worldcubeassociation.org/commit/#{`git rev-parse HEAD`}">Git commit</a>
   <% end %>
```

Comment:
See [a part of this Slack conversation](https://worldcubeassociation.slack.com/archives/i18n/p1487366833925354). I'd suggest something like this
```erb
  <%= alert :info do
    sha1 = `git rev-parse HEAD`
    {
      "PID": Process.pid,
      "Uptime": duration_to_s(Time.now - WcaOnRails::BOOTED_AT),
      "Git commit": link_to(sha1.first(7), "https://github.com/thewca/worldcubeassociation.org/commit/#{sha1}")
    }.map { |key, value| "#{key}: #{value}" }.join(" #{icon "circle"} ").html_safe
  end %>
```
If it's still an overkill in your opinion, I am fine with this code =)